examples/01-blog/Blog/Type/StoryType.php

```diff
@@ -20,10 +18,13 @@ public function __construct()
         parent::__construct([
             'name' => 'Story',
             'fields' => static fn (): array => [
-                'id' => Types::id(),
-                'author' => Types::user(),
-                'mentions' => new ListOfType(Types::mention()),
-                'totalCommentCount' => Types::int(),
+                'id' => ['type' => Types::id(), 'resolve' => static fn (Story $story): int => $story->id],
```

Comment:
should also work with the default resolver

Reply:
342db408